io/pci/distro_tools.py

```diff
@@ -40,6 +40,7 @@ def setUp(self):
         self.option = self.params.get("test_opt", default='')
         self.tool = self.params.get("tool", default='')
         self.pci_device = self.params.get("pci_device", default='')
+        self.adapter_type = self.params.get("adapter_type", default='')
```

Comment:
Why do we need this ?
Can't this be found via `pci.get_pci_class_name()` https://github.com/avocado-framework/avocado/blob/781adbaa7a0736f10114fb8f5bd107f08f495eb5/avocado/utils/pci.py#L138

Reply:
ok i will use this.